package.json

```diff
@@ -32,6 +32,7 @@
     "@enact/ui": "^3.1.2",
     "classnames": "^2.2.6",
     "color-convert": "^1.9.3",
+    "deep-freeze": "0.0.1",
```

Comment:
Should this be a dev-dependency?

Reply:
No. `devDependencies` are for build-time or maintainer dependencies. This is a regular dependency even though it only applies to "dev mode"

Reply:
More concretely, if it were a dev dependency, if an author npm installed agate and tried to `enact serve` their app, it would throw because `deep-freeze` was missing.

Reply:
Makes sense as a regular dependency since it's content being included in builds, rather than a build/test tool :+1: